src/auth/utils.ts

```diff
@@ -50,6 +50,7 @@ export async function getDashboardLoginURL(
  * @returns whether the login was successful or not
  */
 export async function loginIfTokenValid(token?: string): Promise<boolean> {
+	// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
I think that we don't need this and can use `??`, since `if (!token)` right after will preserve the original behavior for any falsy value afaict.
```suggestion
```

Reply:
Reverted to `??`